src/Icicle/Sea/FromAvalanche/Prim.hs

```diff
@@ -165,6 +165,8 @@ seaOfPrimArray p
       -> PDAlloc (prefixOfValType (ArrayT t) <> "put_immutable") Nothing
      PrimArraySwap t
       -> PDFun   (prefixOfValType (ArrayT t) <> "swap") Nothing
+     PrimArrayDel t
+      -> PDFun   (prefixOfValType (ArrayT t) <> "delete") Nothing
```

Comment:
hm, actually, now I'm not sure. Maybe we should have both `delete_immutable` and `delete_mutable`, like for `insert`